modules/service_accounts/main.tf

```diff
@@ -65,7 +65,7 @@ resource "google_project_iam_member" "secretmanager_admin" {
 ####### service account for kms and gcs cross project access
 resource "google_service_account" "kms_gcs_sa" {
   count        = var.create_workload_identity == true ? 1 : 0
-  account_id   = var.kms_gcs_sa_name
+  account_id   = substr("kms-gcs-${random_id.main.hex}", 0, 30)
```

Comment:
could we pass in the customer name here? (so that we can identify which service account belongs to which customer).

Reply:
account_id   = substr("stackdriver-${var.customer_name}-${random_id.main.hex}", 0, 30)

Reply:
This should be resolved. The `random_id` has the [var.namespace](https://github.com/wandb/terraform-google-wandb/pull/144/files#diff-33389c5e33c24b5344a73686778f179cf4aab61d2ec481e590b8db73f1363548R8) in it.